README.md

```diff
@@ -23,23 +23,40 @@ Get started with Voice on Android:
 
 To get started with the Quickstart application follow these steps. Steps 1-6 will allow you to make a call. The remaining steps 7-8 will enable push notifications using FCM.
 
-1. [Open this project in Android Studio](#bullet1)
-2. [Create a Voice API key](#bullet2)
-3. [Configure a server to generate an access token to use in the app](#bullet3)
-4. [Create a TwiML application](#bullet4)
-5. [Configure your application server](#bullet5)
-6. [Run the app](#bullet6)
-7. [Generate google-services.json](#bullet7)
+1. [Open this project in Android Studio](#bullet1)
```

Comment:
You could make the case for this being in step 1. since currently step 1 would result in a gradle error since it's missing the `google-services.json` file. Maybe it's worth noting that.

Reply:
@aaalaniz what do you think?

Reply:
I chose to make it step #2 because of Firebase Assistant support in AS (https://firebase.google.com/docs/android/setup#assistant)

Reply:
I vote step 1. I don't like the first thing to happen to be an AS error.